test/middleware/empty-response.test.ts

```diff
@@ -0,0 +1,46 @@
+import { Context, Response } from 'koa';
+import emptyResponse from '../../src/middleware/empty-response';
+import createContext from '../context';
+import runMiddleware, { Next } from '../middleware';
+
+const makeRequest = async (next?: Next): Promise<Response> => (
+  runMiddleware(emptyResponse(), createContext(), next)
+);
+
+describe('Empty response middleware', (): void => {
+  it('sets an empty string body if it has not been set', async (): Promise<void> => {
+    const response = await makeRequest();
+
+    expect(response.body).toBe('');
+  });
+
+  it('removes redundant headers if no body has been set', async (): Promise<void> => {
+    const next = async ({ response }: Context): Promise<void> => {
+      response.set('Content-Length', '100');
+      response.set('Content-Type', 'content/type');
+      response.set('Transfer-Encoding', 'identity');
+    };
+
+    const response = await makeRequest(next);
+
+    expect(response.get('Content-Length')).toHaveLength(0);
```

Comment:
any perceivable difference between the header being missing and having length 0?

Reply:
`Koa.Response` returns an empty string, would like to find a better way to check the header doesn't exist at all.